migration/migrator/migrations/course/20230609110652_display_pronouns.py

```diff
@@ -0,0 +1,38 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    sql = """ALTER TABLE users ADD COLUMN IF NOT EXISTS display_pronouns
+      boolean DEFAULT false;"""
+    database.execute(sql)
+    pass
+
+
+def down(config, database, semester, course):
+    """
+    Run down migration (rollback).
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    sql = """ALTER TABLE users DROP COLUMN IF EXISTS display_pronouns;"""
+    database.execute(sql)
```

Comment:
We generally don't want to delete any data in the down migration in case something gets rolled back temporarily and then rolled forward again shortly thereafter.